.release-version

```diff
@@ -1 +1 @@
-1.22.0
+1.22.1
```

Comment:
Hmm I wouldn't classify this a bug fix but it's not a feature either. Probably closer to a feature than a bug.